gradlew

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
+# Copyright 2015 the original author or authors.
```

Comment:
Do we need to update the gradlew in all the other projects, too?

Reply:
Or is that because of 5.3.1 -> 5.4.1 vs 5.2.1 -> 5.4.1

Reply:
there is only one gradle project

Reply:
yes there other versions where not updated when in xtext
thus here the scripts are changed too